ThreeBotPackages/tfgrid_solutions/tfgrid_solutions/chatflows/domain_delegation.py

```diff
@@ -1,57 +1,75 @@
 from Jumpscale import j
 
 
-def chat(bot):
+class DomainDelegation(j.servers.chatflow.get_class()):
     """
     """
+
+    steps = ["domain_name", "expiration_time", "domain_pay", "success"]
     user_form_data = {}
```

Comment:
this can not be done on the class instance you need to initialize it in the `__init__` of you instance instead